src/NUnitTestAdapter/VSTest2NUnitFilterConverter.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NUnit.VisualStudio.TestAdapter
+{
+    public class VsTest2NUnitFilterConverter
+    {
+        private string NUnitFilter { get; }
+        public VsTest2NUnitFilterConverter(string vstestfilter)
+        {
+            NUnitFilter = vstestfilter.Replace("TestCategory", "Category");
+        }
```

Comment:
This might just be personal preference but doing the heavy lifting of the class doesn't feel very "constructory" to me.
The name is `Converter` but once you have an instance, you can't convert things with it. You can only access a conversion result.
What would you think of introducing a static `string (string)` method?